selfhost/typechecker.jakt

```diff
@@ -2012,6 +2012,37 @@ struct Typechecker {
         }
     }
 
+    function statement_definitely_returns(this, anon statement: CheckedStatement) -> bool => match statement{
+        Return(expr) => true
+        // FIXME: CheckedStatement::If branches ugly implementation due to current compiler limitations
```

Comment:
It would be nice if this comment told us *which* limitation is making the code ugly, so that we'll know when to fix it :)